src/ResourceManager/CognitiveServices/Commands.Management.CognitiveServices/packages.config

```diff
@@ -1,4 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Azure.Management.CognitiveServices" version="4.0.0" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.CognitiveServices" version="4.1.0" targetFramework="net452" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.13" targetFramework="net452" />
```

Comment:
@yangyuan same comment about removing the extra references in this file

Reply:
Done